### ISSUE TITLE
GET$URL

### ISSUE THREAD

**Issue by Cyberavater** [timmyrs/Universal-Bypass#137]
Example link: http://gslink.co/ePuL

Can you make Universal-Bypass work on GET$URL or at least create a function so that it clicks on "Submit" & "Skip Ad" automatically?

**Comment by Cyberavater** [comment]
Well, it automatically clicks on  "Submit" now but after that, I have to click on "Skip Ad" manually. Can't it click on both accordingly?

**Comment by timmyrs** [comment]
Weird, it does click Skip Ad automatically for me. Are you sure you're using Universal Bypass 9.6.1?

**Comment by Cyberavater** [comment]
Yes, before 9.6.1 it didn't even click on "Submit" let alone "Skip Ad".

**Comment by Cyberavater** [comment]
Yup, I rechecked it anyway. It's 9.6.1

**Comment by timmyrs** [comment]
It clicks "Submit" and "Skip Ad" for me on Firefox and Chrome using your provided example link.

**Comment by Cyberavater** [comment]
Yes, It is clicking on both in firefox for me but not on Chrome. I wonder why?

**Comment by Cyberavater** [comment]
In Chrome, however, the page loads infinitely.

**Comment by timmyrs** [comment]
Do you have some extension which might be interfering on Chrome?

**Comment by Cyberavater** [comment]
Yes, It worked after disabling all other extension.

**Comment by timmyrs** [comment]
Maybe you can narrow it down to one extension, so I can see why it's breaking the bypass.

**Comment by Cyberavater** [comment]
Yes, I'm already on it might need some minute.

**Comment by Cyberavater** [comment]
Found It!
IDM Integration Module -6.32.6

**Comment by timmyrs** [comment]
That's weird.

**Comment by Cyberavater** [comment]
Well, I rechecked.

**Comment by Cyberavater** [comment]
Wait, I'm linking it right now.

**Comment by Cyberavater** [comment]
https://nofile.io/f/ty1vfEAo86j/IDMGCExt.crx

**Comment by Cyberavater** [comment]
I uploaded it from my IDM folder. It should be virus free.

**Comment by timmyrs** [comment]
I found the extension on the Chrome Web Store.

**Comment by Cyberavater** [comment]
Okay.

**Comment by Cyberavater** [comment]
Last I checked it wasn't there I recently didn't check tho.

**Comment by timmyrs** [comment]
The code of that extension is obfuscated, so I can't see what exactly it's doing.

**Comment by Cyberavater** [comment]
But, U can reproduce the error with IDM Integration Module, right?

**Comment by timmyrs** [comment]
I'm not installing a closed-source, obfuscated extension.

**Comment by Cyberavater** [comment]
Oh, I see. Anyway thanks for trying n all.

**Comment by Cyberavater** [comment]
IMO, since 10,000,000+ users on chrome web store use this extension (IDM Integration Module) it wouldn't be a waste if you could solve the problem.

**Comment by timmyrs** [comment]
Just installed the extension and Universal Bypass still bypassed gslink.co. Perhaps another extension is at fault?

**Comment by Cyberavater** [comment]
I rechecked using only IDM Integration Module n Universal Bypass. It still fails. Perhaps I should give you Screenshot of console if that's any helpful?

**Comment by timmyrs** [comment]
Sure.

**Comment by Cyberavater** [comment]
https://imgur.com/a/VLuvbyJ

**Comment by timmyrs** [comment]
Sadly, that's not giving me any new information.

**Comment by Cyberavater** [comment]
Hmm... I'm not a coder as you've already guessed talking with me. So, I don't know exactly what information you might be needing to solve this. I suggested it because nano adblocker asks for it when reporting an error.

**Comment by timmyrs** [comment]
Yeah, that's usually useful in the case of errors, but there were no errors in the console, so it doesn't help in this case.

**Comment by Cyberavater** [comment]
I see, since you can't reproduce the error it would be impossible for you to solve the error. Leaving it, for the time being, should be wise. Meanwhile, I'll try to find related information. I'll report back if I find something interesting.

**Comment by Cyberavater** [comment]
Seems like they added captcha turn this undo this update.
Example Link: http://gslink.co/fzef

**Comment by vungsung** [comment]
Recently they added a new format like the one Cyberavater mentioned (with Captcha). But they had an older format which was like:

[http://gslink.co/a/4890/aHR0cDovL2dzbGluay5jby9hLzQ4OTAvYUhSMGNEb3ZMM1JvWlc1bGQzTmpZWE4wY3k1amIyMHZkbWxsZHk4eWRtUjM=](http://gslink.co/a/4890/aHR0cDovL2dzbGluay5jby9hLzQ4OTAvYUhSMGNEb3ZMM1JvWlc1bGQzTmpZWE4wY3k1amIyMHZkbWxsZHk4eWRtUjM=)

This format is even used a lot by some websites now.
They are very easy to decode, just decode the latter string like
`"aHR0cDovL2dzbGluay5jby9hLzQ4OTAvYUhSMGNEb3ZMM1JvWlc1bGQzTmpZWE4wY3k1amIyMHZkbWxsZHk4eWRtUjM="`
for twice with base64, and then we can get the original URL.

Some codes:
```
url = "http://gslink.co/a/4890/aHR0cDovL2dzbGluay5jby9hLzQ4OTAvYUhSMGNEb3ZMM1JvWlc1bGQzTmpZWE4wY3k1amIyMHZkbWxsZHk4eWRtUjM="
url = url.substr(url.lastIndexOf('/') + 1) #"aHR0cDovL2dzbGluay5jby9hLzQ4OTAvYUhSMGNEb3ZMM1JvWlc1bGQzTmpZWE4wY3k1amIyMHZkbWxsZHk4eWRtUjM="
url = atob(url) #"http://gslink.co/a/4890/aHR0cDovL3RoZW5ld3NjYXN0cy5jb20vdmlldy8ydmR3"

//Can be replaced by For loop
url = url.substr(url.lastIndexOf('/') + 1) #"aHR0cDovL3RoZW5ld3NjYXN0cy5jb20vdmlldy8ydmR3"
url = atob(url) #"http://thenewscasts.com/view/2vdw" #Got the original URL.
```

Could you make this older format bypass, please? Thanks!

**Comment by Cyberavater** [comment]
Can you add crowd-bypass for this domain as well (along with the already added update)?

**Comment by timmyrs** [comment]
It looks like they're using the AdLinkFly Template, which is bypassed by Crowd Bypass.

**Comment by Cyberavater** [comment]
Maybe, but I'm not seeing Crowd-Bypass working there.
Example Link: http://gslink.co/fzef

**Comment by timmyrs** [comment]
Seems like there's only a captcha, and not a timer. And, the final redirect is indirect, so making a crowd bypass for that might be a bit more complicated. Although useful? I'm not sure how popular that site is.

**Comment by Cyberavater** [comment]
Crowd-Bypass for that site might prove useful (IMO). If it's complicated maybe ignoring it for time being might be wise. Just keep it open and let's see if it's popular.

**Comment by timmyrs** [comment]
Yeah, that was my plan.
